pkgs/by-name/wa/waybar/package.nix

```diff
@@ -45,7 +45,6 @@
   wayland-scanner,
   wireplumber,
   wrapGAppsHook3,
-
   cavaSupport ? true,
```

Comment:
I would keep this space as it improves separation between dependencies and "options".